frontend/src/app/Settings/Users/UserDetail.tsx

```diff
@@ -108,8 +127,30 @@ const UserDetail: React.FC<Props> = ({
           />
         ) : null}
         {user.status !== "SUSPENDED" &&
-        user.status !== "PROVISIONED" &&
-        user?.id !== loggedInUser.id ? (
+          user.status !== "PROVISIONED" &&
+          user?.id !== loggedInUser.id ? (
```

Comment:
It might be worth breaking this out to a new "is user active" method, since we're using it in a few spots now

Reply:
Good idea - done!